### DISCUSSION TITLE
An exception has been thrown during the rendering of a template

### DISCUSSION THREAD

**Discussion by Skeeze-Ord** [OFFLINE-GmbH/oc-boxes-support#128] [Help] [answered]
The name of the box matches the documentation. What could be the problem?
![image](https://github.com/OFFLINE-GmbH/oc-boxes-support/assets/58340647/6a17bc3d-170b-471d-8fa8-02510dd5ac06)
![image](https://github.com/OFFLINE-GmbH/oc-boxes-support/assets/58340647/8d8738ce-56cb-4344-a322-a1a03ea2168f)

**Comment by tobias-kuendig** [comment]
Check your `slider-1-box.htm` you seem to have an error in there.

**Reply by Skeeze-Ord** [reply]
"Error in file name: C:/OSPanel/domains/sells-site.loc/plugins/sells/slboxes/partials/slider-1-box.htm. File names can only contain Latin letters, numbers, underscores and periods. Example of correct file names: page.htm, page, subdirectory/page"

The error indicates that the problem is in the name, and not inside the file. There are no problems inside the file

**Reply by tobias-kuendig** [reply]
How did you register these boxes from your plugin directory? Did you do it according to the docs: https://docs.boxes.offline.ch/use-cases/third-party-boxes.html?

Does setting `RESTRICT_BASE_DIR=false` in your `.env` change the behaviour?

**Comment by Skeeze-Ord** [comment, marked as answer]
I solved the problem. I just changed the plugin version in the composer.json file
